src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -71,6 +71,11 @@ internal class DurableClient : IDurableClient,
 
         string IDurableEntityClient.TaskHubName => this.TaskHubName;
 
+        public override string ToString()
```

Comment:
I realize this might be trivial but, since this is a public method, shouldn't it at least inherit its documentation?

Reply:
yes, it should. Will fix.